ark/type/__tests__/keywords/string.test.ts

```diff
@@ -43,6 +43,17 @@ contextualize(() => {
 		attest(b64url("fn5+").toString()).equals(
 			'must be base64url-encoded (was "fn5+")'
 		)
+
+		const b64parse = type("string.base64.parse")
```

Comment:
Let's move this group of tests for `base64` to its own file like `ip.test.ts` or similar.